MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -89,6 +98,34 @@ public class AudioFileProber
                 Fetch(item, result, cancellationToken);
             }
 
+            string ffmpeg_path = _serverConfigurationManager.GetEncodingOptions().EncoderAppPathDisplay;
+            string args = "-hide_banner -i \"";
+            args += path;
+            args += "\" -af ebur128=framelog=verbose -f null -";
+
+            ProcessStartInfo ffmpeg = new ProcessStartInfo(ffmpeg_path);
+
+            System.Diagnostics.Process proc = new System.Diagnostics.Process();
```

Comment:
use a `using` statement so this gets properly disposed

Reply:
i have added this with latest commit, please let me know if ok